.lintr

```diff
@@ -0,0 +1,20 @@
+linters: linters_with_tags(
+    tags = NULL, # include all linters
+    object_name_linter = NULL,
+    undesirable_function_linter = NULL,
+    implicit_integer_linter = NULL,
+    extraction_operator_linter = NULL,
+    todo_comment_linter = NULL,
+    function_argument_linter = NULL,
```

Comment:
I remember reasons for a few, while the rest are copied from {packagetemplate} (this file needs to be updated with that one anyway, but all these are present in that repo).

1. `object_name_linter`: IIRC, when object names are too long,
2. `undesirable_function_linter`: do not recall, likely does not affect this package,
3. `implicit_integer_linter`: to allow integers passed as `1` instead of forcing `1L` (not important for this package any longer),
4. `extraction_operator_linter`: to allow using `$` IIRC,
5. `todo_comment_linter`: to keep these in as notes to ourselves. Will aim to remove.
6. `function_argument_linter`: I believe to allow args with default values to be at any position. Needed primarily for `estimate_ascertainment`